### PR TITLE
Group toolbar buttons for newsfeeds

### PR DESCRIPTION
**Pull request by chmst**
### Summary of Changes
This PR continues the grouping of action buttons in the backend (see #22832, #22393, #23337 and #23336, #23374, #23368, #23380). This time the reditect links

### Testing Instructions
Go to the newsfeeds in the backend and check the toolbar. There should be a "Change status" dropdown now which activates when you select a redirect link.

### Additional Information
Performing "empty trash" is not possible: https://issues.joomla.org/tracker/joomla-cms/23403

### CODE REVIEW

File: administrator/components/com_newsfeeds/View/Newsfeeds/HtmlView.php [hunk 1]
```diff
@@ -129,55 +129,66 @@ protected function addToolbar()
...
-			$dhtml = $layout->render(array('title' => $title));
-			$bar->appendButton('Custom', $dhtml, 'batch');
+		{	
+			$toolbar->popupButton('batch')
+				->text('JTOOLBAR_BATCH')
+				->selector('collapseModal')
+				->listCheck(true);
 		}
-
+
```
Comment (Quy) [inline administrator/components/com_newsfeeds/View/Newsfeeds/HtmlView.php]: Remove tabs.

File: administrator/components/com_newsfeeds/View/Newsfeeds/HtmlView.php [hunk 2]
```diff
@@ -129,55 +129,66 @@ protected function addToolbar()
...
+			$toolbar->addNew('newsfeed.add');
 		}
 
-		if ($canDo->get('core.edit.state'))
-		{
-			ToolbarHelper::publish('newsfeeds.publish', 'JTOOLBAR_PUBLISH', true);
-			ToolbarHelper::unpublish('newsfeeds.unpublish', 'JTOOLBAR_UNPUBLISH', true);
-			ToolbarHelper::archiveList('newsfeeds.archive');
-		}
+		if ($canDo->get('core.edit.state') || Factory::getUser()->authorise('core.admin'))
```
Comment (Quy) [inline administrator/components/com_newsfeeds/View/Newsfeeds/HtmlView.php]: `Factory::getUser()->authorise('core.admin')` should be the same as `$user->authorise('core.admin')`? Change below also.

File: administrator/components/com_newsfeeds/View/Newsfeeds/HtmlView.php [hunk 3]
```diff
@@ -129,55 +129,66 @@ protected function addToolbar()
...
-		if ($canDo->get('core.admin'))
-		{
-			ToolbarHelper::checkin('newsfeeds.checkin');
+			$childBar = $dropdown->getChildToolbar();
+
+			$childBar->publish('newsfeeds.publish')->listCheck(true);
+			$childBar->unpublish('newsfeeds.unpublish')->listCheck(true);
+			$childBar->archive('newsfeeds.archive')->listCheck(true);
+
+			if (Factory::getUser()->authorise('core.admin'))
```
Comment (Quy) [inline administrator/components/com_newsfeeds/View/Newsfeeds/HtmlView.php]:
```suggestion
			if ($user->authorise('core.admin'))
```

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on 061707335cf3e73abc367d01232018267a9080dc<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23404">issues.joomla.org/tracker/joomla-cms/23404</a>.</sub>

**Comment by PhocaCz** [comment]
I have tested this item :white_check_mark: successfully on d3dad9763b17fca6b71df23dd9ad2954de64ff84<br /><br />Tested on Joomla! 4.0.0-alpha7-dev


OK<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23404">issues.joomla.org/tracker/joomla-cms/23404</a>.</sub>

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on d3dad9763b17fca6b71df23dd9ad2954de64ff84<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23404">issues.joomla.org/tracker/joomla-cms/23404</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/23404">issues.joomla.org/tracker/joomla-cms/23404</a>.</sub>

**Comment by wilsonge** [comment]
Thankyou :)

**Pull request merged**